IreAy0/notary-dashboard
# updates

### _Story Headline_

**What does this pull request do?**

- x
- y
- z

* Improvements/Refactoring (Optional):
  - etc.

**Additional Notes**
Optional. caveats, alternative strategies that didn't work out, anything else.

- Looking For Feedback

  - file/method
  - implementation, etc.

- Documentation
  - link to relevant docs, highlight readme changes etc.

**Testing Instructions**

- check out this branch and [related branch](branch_url)
- installation instructions
- pages to view, etc,
